include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -241,7 +241,8 @@ class MenuGame
         Discord::discord_manager& mDiscordManager, HGAssets& mAssets,
         HexagonGame& mHexagonGame, ssvs::GameWindow& mGameWindow);
 
-    void init(bool mErrored);
+    void init(bool mErrored, const std::string& level = "");
```

Comment:
Not a fan of this interface, because it relies on `level == ""` as a magic value to avoid loading a command-line level. We should have two separate functions here:

```cpp
void initWithLevel(bool mErrored, const std::string& level);
void initWithoutLevel(bool mErrored); 
```